quiffen/core/transaction.py

```diff
@@ -395,7 +396,7 @@ def from_list(
                         field_info.split(" ")[0]
                     )
             elif line_code in {"T", "U"}:
-                amount = utils.parse_decimal(field_info)
+                amount = utils.parse_decimal(field_info, locale=locale)
```

Comment:
The `utils.parse_decimal` function doesn't take a `locale` argument, so this change won't do anything